contents/docs/feature-flags/bootstrapping.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Client-side bootstrapping
+sidebar: Docs
+showTitle: true
+availability:
+    free: none
+    selfServe: full
+    enterprise: full
+---
+
+> **Note:** Bootstrapping feature flags is only available in our [JavaScript web](/docs/libraries/js) and [React Native](/docs/libraries/react-native) SDKs.
+
+Since is a delay between initializing PostHog and fetching feature flags, feature flags are not always available immediately.This can be detrimental if you want to do something like redirecting to a different page based on a feature flag.
```

Comment:
```suggestion
Since there is a delay between initializing PostHog and fetching feature flags, feature flags are not always available immediately. This can be detrimental if you want to do something like redirecting to a different page based on a feature flag.
```